### DISCUSSION TITLE
Data unit presented in Disks Tab

### DISCUSSION THREAD

**Discussion by struthio** [moosefs/moosefs#465] [Q&A]
hi,

What Unit is presented in Disks tab in column for "Transfer Reads/Writes" ?
I have ex. values [652 MiB/s] [726 MiB/s], which would suggest transfer above 650 mega-bytes-per-second. But the drives are not capable of such speeds. So what information are shown on Disks tab ?

**Comment by chogata** [comment]
Hi,
Those are read and write speed, but as reported by kernel. Remember, that the kernel has cache. So anything over physical capabilities of you hard drives is a result of cache usage.
